docs/pages/repo/docs/handbook/tools/storybook.mdx

```diff
@@ -0,0 +1,192 @@
+
+  </Tab>
+  <Tab>
+    ```shell
+    cd workshop
+    npx storybook init
+    ```
+  </Tab>
+  <Tab>
+      <p className="nx-mt-6 nx-leading-7">If you're using `pnpm`, you'll need to add an `.npmrc` at the root of your monorepo:</p>
+      ```txt filename=".npmrc"
+      auto-install-peers=true
+      legacy-peer-deps=true
+      node-linker=hoisted
+      ```
+      Then, we scaffold Storybook and install its dependencies manually:
+
+      ```shell
+      cd workshop
+      pnpx sb init -s            # Set up files
+      pnpm i -D @storybook/cli   # Manually install deps and CLI
+      ```
+
+  </Tab>
+</Tabs>
+
+### 3. Set up a story for your Button component
+
+The Storybook scaffold creates some stories and React components in the `/src/stories` directory. To create a story for the button from your `ui` package, we will replace the import in `Button.stories.tsx` with our own.
+
+1. Update the Button in your `ui` package to match the story's specifications.
+
+```jsx filename="packages/ui/Button.tsx"
+interface Props {
+  primary?: boolean;
+  size?: "small" | "large";
+  label?: string;
+}
+
+export const Button = ({
+  primary = false,
+  label = "Boop",
+  size = "small",
+}: Props) => {
+  return (
+    <button
+      style={{
+        backgroundColor: primary ? "red" : "blue",
+        fontSize: size === "large" ? "24px" : "14px",
+      }}
+    >
+      {label}
+    </button>
+  );
+};
+```
+
+2. Add your `ui` package to the `workshop` app:
+
+<Tabs items={["npm", "yarn", "pnpm"]} storageKey="selected-pkg-manager">
+  <Tab>
+```json filename="apps/workshop/package.json"
+{
+  // ...
+  {
+    "dependencies": {
+      "ui": "*",
+      // ...
+    }
+  }
+}
+```
+
+And `npm install` one more time to make sure that your `ui` package is installed in the `workshop` app.
+
+  </Tab>
+  <Tab>
+```json filename="apps/workshop/package.json"
+{
+  // ...
+  {
+    "dependencies": {
+      "ui": "*",
+      // ...
+    }
+  }
+}
+```
+
+And `yarn install` to make sure that your `ui` package is installed in the `workshop` app.
+
+  </Tab>
+  <Tab>
+```json filename="apps/workshop/package.json"
+{
+  // ...
+  {
+    "dependencies": {
+      "ui": "workspace:*",
+      // ...
+    }
+  }
+}
+```
+And `pnpm install` one more time to make sure that your `ui` package is installed in the `workshop` app.
+  </Tab>
+</Tabs>
+
+3. Replace the `Button` import in the `Button.stories.tsx` so that it comes from your `ui` package:
+
+```jsx filename="apps/workshop/src/stories/Button.stories.tsx"
+import { Button } from 'ui'
+```
+
+### 3. Align tasks
+
+The last thing that we need to do is make sure that Storybook is lined up with the rest of your tasks:
+
+```json filename="apps/workshop/package.json"
+{
+  // ...
+  "scripts": {
+    "dev": "start-storybook -p 6006",
+    "build": "build-storybook"
+  }
+}
+```
+
+To ensure build caching, you'll also need to add `storybook-static` to your `.gitignore`.
```

Comment:
We'll need to add another step here to add this output to your turbo.json so it does get cached. Example:
https://github.com/vercel/turbo/blob/main/examples/design-system/turbo.json#L5